react/features/base/tracks/middleware.js

```diff
@@ -326,6 +337,23 @@ function _handleNoDataFromSourceErrors(store, action) {
     }
 }
 
+function createFakeScreenShareParticipant(store, action) {
+    const { track } = action;
+    const { dispatch } = store;
+    const { conference } = store.getState()['features/base/conference'];
+    const particiipantId = track?.jitsiTrack?.getParticipantId?.();
+    const particiipant = getParticipantById(store.getState(), particiipantId);
+    const name = particiipant?.name ? `${particiipant.name}'s screen` : 'Fellow Jitster\'s Screen';
+
+    dispatch(participantJoined({
+        conference,
+        id: track.jitsiTrack.getSourceName(),
```

Comment:
I've decided to use `sourceName` as the `participantId` because it should always be unique and we can parse out the `endpointId`. This can also make it easier to clean up code  on TRACK_REMOVED